apps/iris/src/loader/postgres.go

```
@@ -58,137 +58,41 @@ func parseDatabaseURL(databaseURL string) (string, error) {
-	rows, err := p.client.QueryContext(ctx, selectQuery, key)
+  `, key)
 	if err != nil {
-		p.logger.Log(
-			logger.ERROR,
-			fmt.Sprintf("sql.get.failed stage=query problem_id=%s err=%v", key, err),
-		)
 		return nil, fmt.Errorf("failed to get key: %w", err)
 	}
 
-	defer rows.Close() //nolint:errcheck
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)